src/functions/template-tags/month.php

```diff
@@ -62,15 +62,24 @@ function tribe_show_month( $args = array(), $template_path = 'month/content' ) {
 	/**
 	 * Month view conditional tag
 	 *
-	 * Returns true if the current view is Month
+	 * Returns true when on the "real" Month View itself, but not in other secondary instances of the
+	 * Month View like instance of the [tribe_events] shortcode.
 	 *
 	 * @return bool
 	 */
 	function tribe_is_month() {
 		$tribe_ecp = Tribe__Events__Main::instance();
-		$output    = ( $tribe_ecp->displaying == 'month' ) ? true : false;
-
-		return apply_filters( 'tribe_is_month', $output );
+		$is_month  = ( $tribe_ecp->displaying == 'month' ) ? true : false;
+
+		/**
+		 * Allows filtering of the tribe_is_month boolean value.
+		 *
+		 * @since TBD
```

Comment:
On this Since we should say what as added, since we already have this filter in place before this release.